util/src/Pos/Util/Wlog/Compatibility.hs

```diff
@@ -228,9 +228,9 @@ loggingHandler = unsafePerformIO $ do
 
 -- | setup logging according to configuration @LoggerConfig@
 --   the backends (scribes) will be registered with katip
-setupLogging :: MonadIO m => LoggerConfig -> m ()
-setupLogging lc = liftIO $
-    modifyMVar_ loggingHandler $ const $ Log.setupLogging lc
+setupLogging :: MonadIO m => Text -> LoggerConfig -> m ()
+setupLogging cfoKey lc = liftIO $
+    modifyMVar_ loggingHandler $ const $ Log.setupLogging cfoKey lc
```

Comment:
I presume there is a reason why it's not possible to do:
```
loggingHandler :: Text -> LoggerConfig -> MVar LoggingHandler
loggingHandler cfoKey lc = unsafePerformIO $ newMVar $ Log.setupLogging cfoKey lc
```
:man_shrugging:

Reply:
right, we store in this global MVar the 'LoggingHandler';
it will be accessed from other places, so needs to exist.